src/components/atoms/M/M.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { DEFAULT_MESSAGES } from "assets/messages";
+
+const findMessage: (key: string) => string = (key) => DEFAULT_MESSAGES[key];
```

Comment:
A place to add more files containing messages and/or filter based on environment etc.